### PR TITLE
perf: Add scripts for CKF timing test

### PR DESCRIPTION
**Pull request by XiaocongAi**
This PR updates the script for the KF timing test. It also adds the scripts for running the CKF timing test.

### CODE REVIEW

**Comment by asalzburger** [comment]
Hi, 

I will give that a go report back if it works.

File: Examples/Scripts/CKF_timing_vs_mu.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import ROOT
```
Comment (asalzburger) [inline Examples/Scripts/CKF_timing_vs_mu.py:1]: Is ROOT needed here?

File: Examples/Scripts/KF_timing.sh [hunk 2]
```diff
@@ -51,20 +71,21 @@ echo "Number of tracks per event: ${numTracksPerEvent}"
...
 time_stamp=`date +%F%T`
-exe_dir=$PWD
+exe_dir=${PWD}
 run_dir=KalmanFitter_timing_${time_stamp}
+
 mkdir ${run_dir}
 cd ${run_dir}
 
 output_file='output.log'
-echo "*****KalmanFitter timing test vs. pT*****" > ${output_file}
```
Comment (asalzburger) [inline Examples/Scripts/KF_timing.sh:60]: Would you rather have a `pT` version, I think that would make it clearer, but then we need to ask @msmk0 to change the input, I suppose.

**Review by asalzburger — APPROVED** [review]

**Comment by asalzburger** [comment]
There are two questions, I have also rebased it in the meantime.

File: Examples/Scripts/KF_timing.sh (hunk 2 above)
Comment (XiaocongAi) [reply Examples/Scripts/KF_timing.sh:60]: Exactly. Let's handle this after we have the `pT` input.

File: Examples/Scripts/CKF_timing_vs_mu.py (hunk 1 above)
Comment (XiaocongAi) [reply Examples/Scripts/CKF_timing_vs_mu.py]: No... Removed now.

File: Examples/Scripts/KF_timing.sh (hunk 2 above)
Comment (msmk0) [reply Examples/Scripts/KF_timing.sh:60]: I am not quite sure which input you are referring to here?

Comment (asalzburger) [reply Examples/Scripts/KF_timing.sh:60]: It's the `--gen-p-gev` which was before `--gen-pt-range`, though I agree with @XiaocongAi - we can address this in another PR.

**Review by asalzburger — APPROVED** [review]

**Pull request merged**